presto-main/src/main/java/com/facebook/presto/sql/planner/PlanOptimizers.java

```diff
@@ -79,7 +79,9 @@ public PlanOptimizers(Metadata metadata, SqlParser sqlParser, FeaturesConfig fea
         builder.add(
```

Comment:
can you add an instance of `AbstactTestQueries` which is using new optimizer to unit tests?